database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -100,6 +100,8 @@ class SlickDatabase(override val originalDatabaseConfig: Config) extends SqlData
     actionThreadPool, database.executor.executionContext.reportFailure
   )
 
+  protected[this] lazy val insertBatchSize = databaseConfig.as[Option[Int]]("db.insertBatchSize").getOrElse(2000)
```

Comment:
Aha- glad you mentioned the "db" part. There be dragons messing around with the "db" config sub-namespace. Keys under "db" are actually used by slick and passed into hikari connection polling and the underlying database driver:
https://github.com/slick/slick/blob/3.1.1/slick/src/main/scala/slick/backend/DatabaseConfig.scala#L72-L73

As long as the use of the "db" sub-namespace is intended, everything should be fine. Otherwise, I'd recommend keeping config keys one level up, at `database.insertBatchSize` instead of `database.db.insertBatchSize`.